_site/blog/index.html

```diff
@@ -147,24 +165,6 @@ <h2 class="post-title">Portus 2.1 has been released!</h2>
       </div>
     </div>
     
-    <div class="post-preview col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
```

Comment:
Was this removed because now it goes on the second page? I dunno know how jekyll works. :)

Reply:
No idea. This is done automatically by Jekyll...